<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <link rel="stylesheet" href="style/base.css" type="text/css" />


<!-- // modif : lecorguille -->
    <script src="W4M/jquery-ui-1.11.4.custom/external/jquery/jquery.js"></script>
    <script src="W4M/jquery-ui-1.11.4.custom/jquery-ui.js"></script>
    <link rel="stylesheet" href="W4M/jquery-ui-1.11.4.custom/jquery-ui.css">

    <script>
        $(document).ready(function () {
            $(function() {
                $( "#W4M #accordion > div" ).accordion({ heightStyle: "content",collapsible: true });}
            );
            $(function() {
                $( "#W4M #changelog_accordion" ).accordion({ heightStyle: "content",collapsible: true });
            })
            $(function() {
                $( "#W4M #tutorial_accordion" ).accordion({ heightStyle: "content",collapsible: true });
            })
            /**$("#W4M #tutorial_accordion > div").accordion({ heightStyle: "content",collapsible: true });*/
            $(function() {
                $( "#W4M .collapseclose" ).accordion({ active: false });}
            );
        });
        // fonction toogle for diplay block <-> none
        function toogleDisplay(id)
        {
        if (document.getElementById(id).style.display == "block") {
            document.getElementById(id).style.display = "none";
        }
        else {
            document.getElementById(id).style.display = "block";
        }
        }

    </script>

    <style>

        /*FROM base.css*/
        .errormessagelarge,.warningmessagelarge,.donemessagelarge,.infomessagelarge{padding:15px;margin-bottom:17px;border:1px solid transparent;border-radius:3px;min-height:36px;padding-left:52px;background-image:url(error_large.png);background-repeat:no-repeat;background-position:10px 8px}.errormessagelarge h4,.warningmessagelarge h4,.donemessagelarge h4,.infomessagelarge h4{margin-top:0;color:inherit}
        .errormessagelarge .alert-link,.warningmessagelarge .alert-link,.donemessagelarge .alert-link,.infomessagelarge .alert-link{font-weight:bold}
        .errormessagelarge>p,.warningmessagelarge>p,.donemessagelarge>p,.infomessagelarge>p,.errormessagelarge>ul,.warningmessagelarge>ul,.donemessagelarge>ul,.infomessagelarge>ul{margin-bottom:0}
        .errormessagelarge>p+p,.warningmessagelarge>p+p,.donemessagelarge>p+p,.infomessagelarge>p+p{margin-top:5px}
        .errormessagelarge{background-color:#f9c7c5;border-color:#dd1c15;color:#7e2a27;padding-left:52px}.errormessagelarge hr{border-top-color:#c61913}
        .errormessagelarge .alert-link{color:#571d1b}
        .warningmessagelarge{background-color:#fce1ba;border-color:#e28709;color:#80571e;padding-left:52px;background-image:url(warn_large.png)}.warningmessagelarge hr{border-top-color:#c97908}
        .warningmessagelarge .alert-link{color:#573b14}
        .donemessagelarge{background-color:#aff1af;border-color:#20b420;color:#295f29;padding-left:52px;background-image:url(ok_large.png)}.donemessagelarge hr{border-top-color:#1c9e1c}
        .donemessagelarge .alert-link{color:#193b19}
        .infomessagelarge{background-color:#a6e4f7;border-color:#1197c0;color:#1f5566;background-image:url(info_large.png)}.infomessagelarge hr{border-top-color:#0f85a8}
        .infomessagelarge .alert-link{color:#13343f}

        #W4M  ul {
            list-style-type: none;
            margin-left:0;
        }
        #W4M  ul > li > ul {
            list-style-type: disc;
            margin-left:20px;
        }
        #W4M  ul > li > ul > li > ul {
            list-style-type: square;
            margin-left:20px;
        }

        video {
            padding-top: 15px;
            padding-bottom: 25px;
        }
        .donemessagelarge, .warningmessagelarge, .infomessagelarge {
            padding-top:2px;
            padding-bottom:5px;
        }
        h1, h2 {
            margin-top:8.5px;
        }


    </style>
<!-- // modif fin : lecorguille -->

</head>
<body>
    <div class="document">
        <div width="auto" align="center">
                <img src="W4M/logo-ifb-mono-metabohub_2.1.png" style="max-width:1200px;width:100%;" />
        </div>

        <div class="donemessagelarge">
            <h1>Current version :  3.0</h1>
            <strong>Publication:</strong> Franck Giacomoni, Gildas Le Corguillé, Misharl Monsoor, Marion Landi, Pierre Pericard, Mélanie Pétéra, Christophe Duperier, Marie Tremblay-Franco, Jean-François Martin, Daniel Jacob, Sophie Goulitquer, Etienne A. Thévenot and Christophe Caron (2014). <strong>Workflow4Metabolomics: A collaborative research infrastructure for computational metabolomics</strong>. Bioinformatics  <a href="http://bioinformatics.oxfordjournals.org/content/early/2014/12/18/bioinformatics.btu813" target="_blank">doi:10.1093/bioinformatics/btu813</a><br/><br/>
            <b>Help and support: <a href="mailto:support@workflow4metabolomics.org">support@workflow4metabolomics.org</a></b>
        </div>

        <!--
        <div class="warningmessagelarge">
            <h2>Galaxy Services Maintenance</h2>
            <ul>
                <li>Thursday, March 2nd : 9:00 am to 7:00 pm: The current version of galaxy.workflow4metabolomics.org will be updated (Galaxy framework and some tools). During this operation all W4M Galaxy services will be unavailable. </li>
            </ul>
        </div>
        -->

        <!-- QUELQUES warningmessagelarge PASSES POUR INSPIRATION
            02/03/2017 - The current version of galaxy.workflow4metabolomics.org will be updated (Galaxy framework and some tools). During this operation all W4M Galaxy services will be unavailable.
            08/06/2015 - W4M is now in maintenance. You can face some access problems during reboots.
            29/07/2014 - Some scripts can be disrupted due to problems around R
            22/01/2015 - Maintenance: W4M will be stopped at <blink><strong><font color="red">17:00 TODAY</font></strong></blink> for an expected duration of 1 hour (Update of Galaxy: security and features)
            10/02/2015 - W4M is now in maintenance. You can face some access problems since we are rebooting it.
            10/02/2015 - You may encounter some problems when re-running former datasets, the tools (specially around xcms and CAMERA) became obsolete. You can run new one in the tools panel. We will try to fix this issue as soon as possible.
        -->

        <div class="infomessagelarge">
            <h2>Latest news</h2>
            <ul>
                <li>10/05/2017 - LC-MS: A new tutorial video explain <b>how to</b> run xcmsSet in parallel on single files [<a href="#single_files">link</a>]</li>
                <li>20/04/2017 - Workflow4Metabolomics v3.0 starts today - Check the changelog section below</li>
            </ul>
        </div>


        <!-- QUELQUES infomessagelarge PASSES POUR INSPIRATION
            08/06/2015 - Workflow4Metabolomics v2.1 (2.0 phase 2) starts today - Check the changelog section below
            01/06/2015 - The W4M 2.0 release is presented in the June 2015 MetaboNews Spotlight <a href="http://www.metabonews.ca/Jun2015/MetaboNews_Jun2015.htm#spotlight" target="_blank">[link]</a>
            21/09/2015 - <a href="http://workflow4metabolomics.org/training/W4Mcourse2015" target="_blank">Ecole-chercheurs</a> : Traitement des données métabolomiques sur l'infrastructure online Workflow4Metabolomics (21-25 Sept. 2015) [in French] / Roscoff, France
            19/12/2014 - W4M publication in Bioinformatics is now <a href="http://bioinformatics.oxfordjournals.org/content/early/2014/12/18/bioinformatics.btu813" target="_blank">available</a> - <strong>Workflow4Metabolomics: A collaborative research infrastructure for computational metabolomics</strong>
        -->

        <div id="W4M">
            <div id="accordion">
                <div class="collapseclose">
                <!--<div>-->
                    <h3>Changelog</h3>

                    <div id="changelog_accordion">

                        <h4>3.0.0 - 20/04/2017 </h4>
                            <div>
                                <ul class="changelog">
                                    <li><h5>LC-MS</h5>
                                        <ul>
                                            <li>Preprocessing
                                                <ul>
                                                    <li>UPGRADE - xcms.* (2.1.0): upgrade the xcms version from 1.44.0 to 1.46.0</li>
                                                    <li><b>NEW</b> - xcms.* (2.1.0): The W4M tools will be able now to take as input a single file. It will allow to submit in parallel several files and merge them afterward using "xcms.xcmsSet Merger" before "xcms.group".</li>
                                                    <li><b>BUGFIX</b> - xcms.xcmsSet (2.1.0): the default value of "matchedFilter" -> "Step size to use for profile generation" which was of 0.01 have been changed to fix with the XMCS default values to 0.1</li>
                                                    <li>BUGFIX - xcms.xcmsSet (2.1.0): propose scanrange for all methods</li>
                                                    <li><b>IMPROVEMENT</b> - xcms.group and xcms.fillPeaks (2.1.0): Add an option to export the peak list at this step without have to wait camara.annotate</li>
                                                    <li><b>BUGFIX</b> - xcms.group (2.1.0): the default value of "density" -> "Maximum number of groups to identify in a single m/z slice" which was of 5 have been changed to fix with the XMCS default values to 50</li>

                                                    <li>BUGFIX - CAMERA.annotate (2.2.0): the diffreport ids didn't convert the rt in minute as the other export</li>
                                                    <li>UPDATE - CAMERA.annotate (2.2.0): the settings (digits, convertion in minutes) of the identifiants will no longer modify the native one. Because we want to be conservative and because it can be dangerous for the data integrity during a futur merge of the table, we decide to put those customization in a new column namecustom within the variableMetadata.</li>
                                                    <li>IMPROVEMENT - CAMERA.annotate (2.2.0): add some sections within the form to separate the different parts of the process</li>
                                                    <li><b>IMPROVEMENT</b> - CAMERA.annotate (2.2.0): add the possibility to use defined ruleset</li>
                                                    <li>IMPROVEMENT - CAMERA.annotate (2.2.0): add the possibility to set the MZ digit within the identifiants</li>
                                                    <li>IMPROVEMENT - CAMERA.annotate (2.2.0): is now compatible with merged individual data from xcms.xcmsSet</li>
                                                </ul>
                                            </li>
                                            <li>Annotation
                                                <ul>
                                                    <li>IMPROVEMENT - ProbMetab (1.1.0): add some sections within the form to separate the different parts of the process</li>
                                                    <li>IMPROVEMENT - ProbMetab (1.1.0): is now compatible with merged individual data from xcms.xcmsSet</li>

                                                    <li>BUGFIX - lcmsmatching(3.3.1): Correct a bug while trying to connect to Peakforest for getting the list of chromatographic columns.</li>
                                                    <li>IMPROVEMENT - lcmsmatching(3.3.1): The file database (in-house) field names are now presented in individual choice lists instead of a single text box where you had to insert a very long keys/values string.</li>
                                                    <li>IMPROVEMENT - lcmsmatching(3.3.1): The tool now tries to guess the names of the file database fields, the values of the MS mode column, and the names of the input file columns.</li>
                                                    <li>IMPROVEMENT - lcmsmatching(3.3.1): Allows to select the unit (minutes or seconds) of retention time values inside the input file, but also inside the file database (in-house).</li>

                                                    <li>NEW - Massbank: A search by pseudo-spectra on MassBank is now available. (Version 2017-02-06)</li>
                                                    <li>IMPROVEMENT - Lipidmaps: the tool support now different ionisation mode of submitted data (Version 2016-09-27)</li>
                                                    <li>IMPROVEMENT - HMDB: the tool produces new outputs including compound names (Version 2016-11-28)</li>
                                                </ul>
                                            </li>
                                        </ul>
                                    </li>
                                    <li><h5>GC-MS</h5>
                                        <ul>
                                            <li>Preprocessing
                                                <ul>
                                                    <li>IMPROVEMENT - metaMS:runGC (2.1): full integration of retention indices for sample and database comparison</li>
                                                </ul>
                                            </li>
                                        </ul>
                                        <ul>
                                            <li>Annotation
                                                <ul>
                                                    <li>NEW - Golm Metabolome Database (2016-12-05): A tool to annotate GCMS data (MSP format) with  Golm Metabolome Database.</li>
                                                </ul>
                                            </li>
                                        </ul>
                                    </li>
                                    <li><h5>NMR</h5>
                                        <ul>
                                            <li>Preprocessing
                                                <ul>
                                                    <li>NEW – nmr_preprocessing (PEPS-NMR): Preprocessing of 1D NMR spectra (Galaxy Version 3.1.0)</li>
                                                    <li>NEW - Alignment (2.0.3): The W4M tools will now be able to realign preprocessed spectra, using the CluPA algorithm (Vu et al. 2011)</li>
                                                    <li>IMPROVEMENT - Bucketing (2.0.2): Add “zoomed” representations of bucketed spectra, depending on exclusion zone borders</li>
                                                    <li>IMPROVEMENT - Normalization: corresponds to the old NMR_Annotation tool, now extended to all technologies</li>
                                                </ul>
                                            </li>
                                        </ul>
                                        <ul>
                                            <li>Annotation
                                                <ul>
                                                    <li>UPDATE – Annotation: update of the mathematical algorithm to identify and quantify metabolites</li>
                                                </ul>
                                            </li>
                                        </ul>
                                    </li>
                                </ul>
                            </div>
                            <h4>2.5.1 - 09/03/2016 </h4>
                        <div>
                            <ul class="changelog">
                                <li><h5>LC-MS and GC-MS</h5>
                                    <ul>
                                        <li>UPDATE/IMPROVEMENT - 09/03/16- Normalisation / Determine_batch_correction (2016-03-03): plot layout improvement</li>
                                        <li>UPDATE/IMPROVEMENT - 09/03/16- Normalisation / Batch correction (2.0.3): plot layout improvement</li>
                                        <li>UPDATE/- 09/03/16- Normalisation / Batch correction (2.0.3): Suppression of combined table for simca</li>
                                        <li>UPDATE/IMPROVEMENT - 09/03/16- Normalisation / Batch correction (2.0.3): Addition of a new parameter "Null values"</li>
                                    </ul>
                                </li>
                            </ul>
                        </div>

                        <h4>2.5.0 - 24/02/2016</h4>
                        <div>
                            <ul class="changelog">
                                <li><h5>LC-MS</h5>
                                    <ul>
                                        <li>BUGFIX - Preprocessing / xcms.* (2.0.5) CAMERA.* (2.1.3): <font color="red">better management of errors</font>. Datasets remained green although the process failed</li>
                                        <li>UPDATE - Preprocessing / xcms.* (2.0.5) CAMERA.* (2.1.3): refactoring of internal management of inputs/outputs</li>
                                        <li>UPDATE - Preprocessing / xcms.* (2.0.5) CAMERA.* (2.1.3): refactoring to feed the new summary tool</li>
                                        <li>BUGFIX/IMPROVEMENT</font> - Preprocessing / xcms.xcmsSet (2.0.7): <font color="red">New checking steps around the imported data</font> in order to raise explicte error message before or after launch XCMS: checking of bad characters in the filenames, checking of the XML integrity and checking of duplicates which can appear in the sample names during the XCMS process because of bad characters</li>
                                        <li>BUGFIX/IMPROVEMENT</font> - Preprocessing / xcms.xcmsSet (2.0.7): <font color="red">New step to check and delete bad characters in the XML</font>: accented characters in the storage path of the mass spectrometer</li>
                                        <li>BUGFIX - Preprocessing / xcms.retcor (2.0.5): some pdf remained empty even when the process succeed</li>
                                        <li>BUGFIX - Preprocessing / CAMERA.annotate (2.1.3): the conversion into minutes of the retention time was applied to the diffreport outputs (several conditions)</li>
                                        <li>IMPROVEMENT - Preprocessing / CAMERA.annotate (2.1.3): when there are several conditions, the tool will generate individual datasets (tsv, pdf) instead of a zip file. The usual png (eic, boxplot) will from now be integrated in two pdf.</li>
                                        <li>BUGFIX - 18/01/16 - Preprocessing / xcms.xcmsSet: Some zip files were tag as "corrupt" by R. We have changed the extraction mode to deal with thoses cases.</li>
                                        <li><font color="red">NEW</font> - Preprocessing / xcms.summary (1.0.0): Create a summary of XCMS and CAMERA analysis</li>
                                    </ul>
                                </li>
                            </ul>
                        </div>
                        <h4>2.4.3 - 10/02/2016</h4>
                        <div>
                            <ul class="changelog">
                                <li><h5>All workflows</h5>
                                    <ul>
                                        <li>UPDATE - 10/02/16 - Statistical Analysis / Multivariate (2.3.0) NEW FEATURES: Predictions now available (see the 'Samples to be tested' argument) OPLS(-DA): Predictive and Orthogonal VIP are now computed. Multiclass PLS-DA implemented. MINOR MODIFICATIONS Changes in color palette: black/grey colors for diagnostics and other colors for scores</li>
                                        <li>BUGFIX - 10/02/16 - Normalisation / Batch_correction (2.0.2): all_loess_sample:  minor bug correction in the figure display (previously stopping in the case of missing 'pool' observations).</li>
                                    </ul>
                                </li>
                                <li><h5>LC-MS</h5>
                                    <ul>
                                        <li>BUGFIX - 18/01/16 - Preprocessing / xcms.xcmsSet: Some zip files were tag as "corrupt" by R. We have changed the extraction mode to deal with thoses cases.</li>
                                    </ul>
                                </li>
                            </ul>
                        </div>
                        <h4>2.4.0 - 04/12/2015</h4>
                        <div>
                            <ul class="changelog">
                                <li><h5>LC-MS</h5>
                                    <ul>
                                        <li>NEW - Statitical Analysis / Biosigner: Molecular signature discovery from omics data</li>
                                    </ul>
                                </li>
                            </ul>
                        </div>
                        <h4>2.3.5 - 30/10/2015</h4>
                        <div>
                            <ul class="changelog">
                                <li><h5>LC-MS</h5>
                                    <ul>
                                        <li>NEW - 30/10/15 - Annotation / LC/MS matching: Matching of mz/rt values onto local reference compound database.</li>
                                        <li>BUGFIX - 15/10/15 - Preprocessing / CAMERA.annotate: There was a bug with the CAMERA.annotate (generating a bad dataMatrix (intensities which don't match with the metabolites))</li>
                                        <li>BUGFIX - 09/10/15 - Preprocessing / xcms.xcmsSet: Some users reported a bug in xcms.xcmsSet. The preprocessing stops itself and doesn't import the whole dataset contained in the zip file without warning. But meanwhile, please check your samplemetadata dataset and the number of rows. Sorry about that!</li>
                                        <li>UPDATE - 30/09/15 - Statistical Analysis / Univariate (2.1.1) Internal handling of 'NA' p-values (e.g. when intensities are identical in all samples)</li>
                                        <li>UPDATE - 30/09/15 - Statistical Analysis / Multivariate (2.2.4) Correction in the Galaxy wrapper (in the previous version, the number of predictive components was sometimes set to the maximum by mistake). The regression coefficients are now provided as a new column of the variableMetadata output</li>
                                        <li>UPDATE - 22/09/15 - Statistical Analysis / Multivariate (2.2.3) The default number of permutations is set to 10 (instead of 100) as a compromise to enable both a quick computation and a first hint at model significance</li>
                                    </ul>
                                </ul>
                        </div>
                        <h4>2.2.1 - 18/09/2015</h4>
                        <div>
                            <ul class="changelog">
                                <li><h5>NMR</h5>
                                    <ul>
                                        <li>NEW - Annotation / NMR_Annotation: Annotation of complex mixture NMR spectra and metabolite proportion estimation</li>
                                    </ul>
                                </li>
                                <li><h5>LC-MS</h5>
                                    <ul>
                                        <li>UPDATE - Statistical Analysis / Multivariate (2.2.2): A default of 100 permutations has been set in order to check for overfitting; in addition,'permutation', 'overview', and 'outlier' plots are now displayed by default. Classification is currently implemented for two-class responses only. dataMatrix is not modified by the tool, so it does not appear as an output files.Double cross-validation (advanced computational parameters): 'odd' now refers to train (instead to test) indices  </li>
                                    </ul>
                                </li>
                            </ul>
                        </div>
                        <h4>2.1.0 - 02/06/2015</h4>
                        <div>
                            <ul class="changelog">
                                <li><h5>General</h5>
                                    <ul>
                                        <li>NEW - The W4M workflows will now take as input a zip file to ease the transfer and to improve dataset exchange between tools and users. (See How_to_upload). The previous "Library directory name" is still available but we invite user to switch on the new zip system as soon as possible.</li>
                                        <li>UPDATE - Implementation of a new upload system to use FTP Galaxy features. It will allow to avoid some boring FTP and Library manipulations and to really use Galaxy dataset manager. (Data must be wrapped in a zip file) (See Tutorial -> Upload datasets)</li>
                                    </ul>
                                </li>
                                <li><h5>LC-MS</h5>
                                    <ul>
                                        <li>NEW - Format Conversion / msconvert_multiple_raw: no longer in beta but there are still some troubles with the Bruker format</li>
                                        <li>UPDATE - Preprocessing / xcms.*: new datatype/dataset formats (rdata.xcms.raw, rdata.xcms.group, rdata.xcms.retcor ...) will facilitate the sequence of tools and so avoid incompatibility errors.</li>
                                        <li>UPDATE - Preprocessing / CAMERA.*: new datatype/dataset formats (rdata.camera.positive, rdata.camera.negative, rdata.camera.quick ...) will facilitate the sequence of tools and so avoid incompatibility errors.</li>
                                        <li>UPDATE - Preprocessing / xcms.* CAMERA.*: parameter labels have changed to facilitate their reading.</li>
                                        <li>UPDATE - Preprocessing / CAMERA.annotate: merged with annotateDiffreport. Some parameters are dedicated to experiences with several conditions</li>
                                    </ul>
                                </li>
                            </ul>
                        </div>
                        <h4>2.0.0 - 01/06/2015</h4>
                        <div>
                            <ul class="changelog">
                                <li><h5>LC-MS</h5>
                                    <ul>
                                        <li>UPDATE - Normalisation / Batch correction: signal drift correction based on sample trend (instead of QCs)</li>
                                        <li>NEW - Normalisation / Transformation: logarithm transformation of intensities</li>
                                        <li>NEW - Quality control / Check Format: checking formats of the dataMatrix, sampleMetadata and variableMetadata</li>
                                        <li>NEW - Quality control / Quality Metrics: provides quality metrics and visualization of the dataMatrix</li>
                                        <li>NEW - Annotation / ProbMetab: automatic probabilistic annotation</li>
                                        <li>NEW - Annotation / match_mass: search by masses on local bank</li>

                                    </ul>
                                </li>
                                <li><h5>GC-MS</h5>
                                    <ul>
                                        <li>NEW - Preprocessing / metaMS.runGC: GC-MS data preprocessing using metaMS package</li>
                                        <li>NEW - Annotation / Golm: to query Golm Metabolome</li>
                                    </ul>
                                </li>
                                <li><h5>NMR</h5>
                                    <ul>
                                        <li>NEW - Preprocessing / NMR_Bucketing: functionalities include bucketing and integration</li>
                                        <li>NEW - Normalisation / NMR_Normalization: several normalization methods (Total intensity, quantitative variable and Probabilistic Quotient Normalization).</li>
                                    </ul>
                                </li>
                            </ul>
                        </div>
                    </div>
                </div>
                <div> <!-- class="collapseclose">-->
                    <h3>Tutorials</h3>
                    <div  id="tutorial_accordion">
                        <h4>HOW TO</h4>
                        <div>
                            <table style="width: 100%;" border="1" cellspacing="1" cellpadding="1"><thead><tr><th>CATEGORY</th><th>FIELD</th><th>Description</th><th>PDF</th></tr></thead><tbody><tr><td style="width: 150px;">Galaxy</td><td style="width: 150px;">&nbsp;</td><td style="width: 369px;">Prepare input datasets</td><td style="width: 100px;"><a href="http://abims.sb-roscoff.fr/mypydio/data/public/aaeb31.php?dl=true">Download</a></td></tr><tr><td style="width: 150px;">Galaxy</td><td style="width: 150px;">&nbsp;</td><td style="width: 369px;">Import datasets</td><td style="width: 100px;"><a href="http://abims.sb-roscoff.fr/mypydio/data/public/8880e7.php?dl=true">Download</a></td></tr><tr><td style="width: 150px;">Galaxy</td><td style="width: 150px;">&nbsp;</td><td style="width: 369px;">Cleanup datasets</td><td style="width: 100px;"><a href="http://abims.sb-roscoff.fr/mypydio/data/public/f0fd7c.php?dl=true">Download</a></td></tr><tr><td style="width: 150px;">Galaxy</td><td style="width: 150px;">&nbsp;</td><td style="width: 369px;">Build And Configure A Workflow</td><td style="width: 100px;"><a href="http://abims.sb-roscoff.fr/mypydio/data/public/9bc79d.php?dl=true">Download</a></td></tr><tr><td style="width: 150px;">Galaxy</td><td style="width: 150px;">&nbsp;</td><td style="width: 369px;">Share Histories And Workflows</td><td style="width: 100px;"><a href="http://abims.sb-roscoff.fr/mypydio/data/public/8e8802.php?dl=true">Download</a></td></tr><tr><td style="width: 150px;">Preprocessing</td><td style="width: 150px;">LC-MS</td><td style="width: 369px;">Perform Preprocessing using XCMS</td><td style="width: 100px;"><a href="http://abims.sb-roscoff.fr/mypydio/data/public/084ad1.php?dl=true">Download</a></td></tr><tr><td style="width: 150px;">Quality Control</td><td style="width: 150px;">Common</td><td style="width: 369px;">Format Data For Postprocessing</td><td style="width: 100px;"><a href="http://web11.sb-roscoff.fr/download/w4m/howto/w4m_HowToFormatDataForPostprocessing_v02.pdf">Download</a></td></tr><tr><td style="width: 150px;">Batch correction</td><td style="width: 150px;">LC-MS and GC-MS</td><td style="width: 369px;">Perform Drift And Batch Correction</td><td style="width: 100px;"><a href="http://web11.sb-roscoff.fr/download/w4m/howto/w4m_HowToPerformDriftAndBatchCorrection_v02.pdf">Download</a></td></tr><tr><td style="width: 150px;">Statistics</td><td style="width: 150px;">Common</td><td style="width: 369px;">Perform Univariate Analyzes</td><td style="width: 100px;"><a href="http://web11.sb-roscoff.fr/download/w4m/howto/w4m_HowToPerformUnivariateAnalyzes_v02.pdf">Download</a></td></tr><tr><td style="width: 150px;">Statistics</td><td style="width: 150px;">Common</td><td style="width: 369px;">Perform Multivariate Analyzes</td><td style="width: 100px;"><a href="http://web11.sb-roscoff.fr/download/w4m/howto/w4m_HowToPerformMultivariateAnalyzes_v02.pdf">Download</a></td></tr><tr><td style="width: 150px;">Annotation</td><td style="width: 150px;">LC-MS</td><td style="width: 369px;">Perform LCMS Annotations</td><td style="width: 100px;"><a href="http://web11.sb-roscoff.fr/download/w4m/howto/w4m_HowToPerformAnnotations_v05.pdf">Download</a></td></tr><tr><td style="width: 150px;">Annotation</td><td style="width: 150px;">GC-MS</td><td style="width: 369px;">Use NIST</td><td style="width: 100px;"><a href="http://application.sb-roscoff.fr/download/w4m/howto/w4m_HowToUseNIST_V01.pdf">Download</a></td></tr></tbody></table>

                            <br/><hr/>
                            <h5 id="single_files">Use single files as input and run xcmsSet in parallel</h5>
                            <video style="align:center;max-width:1280px;width:95%;" controls="controls" scale="tofit" >
                                <source src="http://download.workflow4metabolomics.org/docs/170510_galaxy_xcms_dataset_collection.m4v" type="video/mp4">
                            </video>
                            <a href="http://download.workflow4metabolomics.org/docs/170510_galaxy_xcms_dataset_collection.m4v">Download the video</a>
                        </div>
                        <h4>W4M courses 2015</h4>
                        <div>
                                <table style="width: 100%;" border="1" cellspacing="1" cellpadding="1"><thead><tr><th scope="col">W4M courses 2015</th><th scope="col">Author(s)</th><th scope="col">PDF</th></tr></thead><tbody><tr><td>Course presentation</td><td>Christophe Caron, Franck Giacomoni</td><td><a href="/sites/workflow4metabolomics.org/files/files/00_course_presentation_fg-cc_20150918.pdf" target="_blank">Download</a></td></tr><tr><td>Initiation</td><td>Gildas Le Corguillé, Misharl Monsoor, Marion Landi</td><td><a href="/sites/workflow4metabolomics.org/files/files/galaxy-init-1.05_printable.pdf" target="_blank">Download</a></td></tr><tr><td>LC-MS -&nbsp;Pre-processing (xcms)</td><td>W4M Core Team</td><td><a href="/sites/workflow4metabolomics.org/files/files/w4m_xcms_CAMERA_LC-MS.pdf" target="_blank">Download</a></td></tr><tr><td>MS data processing - Filtering and correcting data</td><td>W4M Core Team</td><td><a href="/sites/workflow4metabolomics.org/files/files/MS_data_processing.pdf" target="_blank">Download</a></td></tr><tr><td>Using&nbsp;Galaxy4Metabolomics -&nbsp;W4M table format for&nbsp;Galaxy</td><td>W4M Core Team</td><td><a href="/sites/workflow4metabolomics.org/files/files/w4m_TableFormatForGalaxy_150908.pdf" target="_blank">Download</a></td></tr><tr><td>How to check the format&nbsp;of the 3 .tsv tables</td><td>Etienne Thévenot</td><td><a href="/sites/workflow4metabolomics.org/files/files/w4m_HowToPerformStatisticalAnalysis_Appendix_CheckFormat_150923.pdf" target="_blank">Download</a></td></tr><tr><td>How to perform statistical analysis ? - Introduction</td><td>Etienne Thévenot, Marie Tremblay-Franco, Mélanie Pétéra, Gildas Le Corguillé</td><td><a href="/sites/workflow4metabolomics.org/files/files/w4m_HowToPerformStatisticalAnalysis_1_Introduction_150923.pdf" target="_blank">Download</a></td></tr><tr><td>How to perform statistical analysis ? - Exploratory data analysis</td><td>Etienne Thévenot, Mélanie Pétéra, Jean-François Martin, Gildas Le Corguillé</td><td><a href="/sites/workflow4metabolomics.org/files/files/w4m_HowToPerformStatisticalAnalysis_2_ExploratoryDataAnalysis_150923.pdf" target="_blank">Download</a></td></tr><tr><td>How to perform statistical analysis ? -&nbsp;PCA analysis and&nbsp;(O)PLS(-DA) modelling</td><td>Etienne Thévenot</td><td><a href="/sites/workflow4metabolomics.org/files/files/w4m_HowToPerformStatisticalAnalysis_4_MultivariateModelling_150923.pdf" target="_blank">Download</a></td></tr><tr><td>Les banques d'annotation - Annotation</td><td>Yann Guitton, Franck Giacomoni</td><td><a href="/sites/workflow4metabolomics.org/files/files/W4M_Annotation_banques_annotation.pdf" target="_blank">Download</a></td></tr></tbody></table><p>&nbsp;</p>
                        </div>
                    </div>
                </div>
            </div>
            <hr />
            <div width="auto" align="center">
                <img src="W4M/workflow_all_HD_color_2.0.png" style="max-width:1500px;width:95%;" /><br/>
                <hr />
                <img src="W4M/workflow_community_HD_2.0.png" style="max-width:1000px;width:95%;" />
            </div>
        </div>


        <hr/>
        <a target="_blank" class="reference" href="http://galaxyproject.org/">Galaxy</a> is an open, web-based platform for data intensive biomedical research.  The <a target="_blank" class="reference" href="http://wiki.galaxyproject.org/GalaxyTeam">Galaxy team</a> is a part of <a target="_blank" class="reference" href="http://www.bx.psu.edu">BX</a> at <a target="_blank" class="reference" href="http://www.psu.edu">Penn State</a>, and the <a target="_blank" class="reference" href="http://www.biology.emory.edu/">Biology</a> and <a target="_blank" class="reference" href="http://www.mathcs.emory.edu/">Mathematics and Computer Science</a> departments at <a target="_blank" class="reference" href="http://www.emory.edu/">Emory University</a>.  The <a target="_blank" class="reference" href="http://galaxyproject.org/">Galaxy Project</a> is supported in part by <a target="_blank" class="reference" href="http://www.genome.gov">NHGRI</a>, <a target="_blank" class="reference" href="http://www.nsf.gov">NSF</a>, <a target="_blank" class="reference" href="http://www.huck.psu.edu">The Huck Institutes of the Life Sciences</a>, <a target="_blank" class="reference" href="http://www.ics.psu.edu">The Institute for CyberScience at Penn State</a>, and <a target="_blank" class="reference" href="http://www.emory.edu/">Emory University</a>.</p>

    </div>
</body>
</html>


<script>

    // modif : lecorguille
    // fonction toogle for diplay block <-> none
    function toogleDisplay(id)
    {
    if (document.getElementById(id).style.display == "block") {
        document.getElementById(id).style.display = "none";
    }
    else {
        document.getElementById(id).style.display = "block";
    }
    }
    // fin modif : lecorguille

</script>
